src/test/java/step3/domain/car/NameTest.java

```diff
@@ -0,0 +1,60 @@
+package step3.domain.car;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class NameTest {
+
+    private static final String NameTooLong = "aaaaaa";
+
+    @DisplayName("이름의 길이가 1이상 5이하가 아니라면 Name 생성시 예외를 던진다.")
+    @ParameterizedTest(name = "[{index}] name: {0}")
+    @NullAndEmptySource
+    @ValueSource(strings = {NameTooLong})
+    void create_illegalLength_thrownException(String inputName) {
+        assertThatIllegalArgumentException()
+                .isThrownBy(() -> new Name(inputName))
+                .withMessage(Name.NAME_LENGTH_ERROR_MESSAGE);
+    }
+
+    @DisplayName("Name 생성 정상일 때")
+    @ParameterizedTest(name = "[{index}] inputName: {0}")
+    @ValueSource(strings = {
+            "정상이름",
+            "정"
+    })
+    void create(String stringName) {
+        Name name1 = new Name(stringName);
+        Name name2 = new Name(stringName);
+
+        assertThat(name1).isEqualTo(name2);
```

Comment:
```suggestion
        Name name = new Name(stringName);

        assertThat(name).isEqualTo(new Name(stringName));
```
이런 방법으로 코드를 써봐도 좋을 것 같아요!